src/Extension/StatusRuntime.php

```diff
@@ -35,9 +36,32 @@ public function addStatusService(StatusClassRendererInterface $renderer): void
     }
 
     /**
-     * @param mixed $statusType
+     * @param object|string $object     Object for StatusClassRenderer or string for FlashManager
+     * @param string|null   $statusType Object status type or Sonata flash message type
+     * @param string        $default    Default status class
+     * @param mixed         $statusType
      */
-    public function statusClass(object $object, $statusType = null, string $default = ''): string
+    public function statusClass($object, $statusType = null, string $default = ''): string
+    {
+        if (\is_object($object)) {
```

Comment:
```suggestion
        if ($object instanceof StatusClassRendererInterface) {
```

Reply:
Here `$object` is handles by some `*StatusRenderer` class. `$object is not instanceof `StatusClassRendererInterface`.
```php
class OrderStatusRenderer implements StatusClassRendererInterface
{
    public function handlesObject($object, $statusName = null)
    {
        return ($object instanceof OrderInterface || $object instanceof OrderElementInterface)
            && \in_array($statusName, ['delivery', 'payment', null], true);
    }
    ...
}
```

Reply:
IMO, if we know what union types it could be, I think is better to check against these types instead of relaxing the check just to `object`.

Reply:
Mhh, then, my suggestion was wrong [here](https://github.com/sonata-project/twig-extensions/pull/92/files#diff-05e712f490a9e8405b9e36c2136df8e9R57).
I'm leaving a new suggestion. Sorry for the misunderstanding.